examples/get-started/src/components/AllFlagsUsingHOC.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react'
+import { withFeatureFlags } from '@harnessio/ff-react-client-sdk'
+import Loader from './Loader/Loader'
+
+const AllFlagsUsingHOC: FC = withFeatureFlags(({ flags, loading }) => (
+  <section>
+    <h2>All feature flags</h2>
```

Comment:
nitpick:
```suggestion
    <h2>All Feature Flags</h2>
```

Reply:
Nice attention to detail 👍🏻 I was mixed on this as _Feature Flags_, the product, is a "Harness noun", but in this context we're referring to the flags themselves.

Reply:
okie doke 👍